src/test/java/uk/ashleybye/httpserver/server/ServerTest.java

```diff
@@ -167,6 +210,17 @@ void testPostMethod() {
     assertEquals("HTTP/1.1 200 OK\n\r\nsome body", connection.getSentData());
   }
 
+  @Test
+  void testRedirect() {
```

Comment:
I would recommend using more descriptive test names here. For example, "testRedirectReturnsTheCorrectHttpHeaders" or something like that